packages/cli-tool/src/commands/generate/index.ts

```diff
@@ -1,8 +1,12 @@
-import {Command, Flags, Hook} from '@oclif/core'
-import {ChildProcess} from 'node:child_process'
-import Inquirer from 'inquirer'
 import * as fs from 'node:fs'
 
+import {Command} from '@oclif/core'
+import {cli} from 'cli-ux'
+import Inquirer from 'inquirer'
+
+import getChoices from '../../helpers/choices'
+import {formatHookErrorMsg, hookFailed} from '../../helpers/hook-error'
```

Comment:
I didn't find it in the compass, and I don't have access to other company react projects.

So, we do not frown upon relative imports in our codebase. Is it correct?

Reply:
Thanks, indeed we do not like them either ;P The generated projects from this template includes a config to prevent them! :-) 

I'm trying to add a baseUrl into the tsconfig, but I ended up with the error below. Do you have any idea what it could be? 🤔 

<img width="933" alt="image" src="https://user-images.githubusercontent.com/77609814/177287996-108d96e3-0629-4d53-a65c-faabd7472b5e.png">

Reply:
Hum... got' it... @carryall found this issue: https://github.com/oclif/oclif/issues/288

Natively, OCLIF does not support it. There is a workaround (last post of this issue) but it adds some complexity for little value IMO, so I would recommend keeping the relative imports if that's fine for you~ 

> Note that our [infrastructure template](https://github.com/nimblehq/infrastructure-templates/blob/develop/src/commands/generate/index.ts) also uses relative imports 😇

Reply:
Good for me, just wanted to clarify this moment 👍